modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle

```diff
@@ -0,0 +1,17 @@
+import com.liferay.gradle.util.OSDetector
+
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":with-version:deploy", ":without-version:deploy"
```

Comment:
for the `clientExtensionXXX` tests, most of them follow this pattern: `xyz-client-extension`. lets apply that here too for consistency

Reply:
Done